stdlib/list.gr

```diff
@@ -231,6 +231,25 @@ let rec dropWhile = (fn, list) => {
   }
 }
 
+let rec take = (n, list) => {
+  if (n < 0) {
+    fail 'number of items to take cannot be a negative number'
+  } else {
+    match ((n, list)) {
+      | (_, []) => list
+      | (n, _) when n == 0 => []
```

Comment:
Could match on `(0, _)` here but I think this is more obvious/clear.

Reply:
Yeah, you absolutely right. The syntax is not that familiar for me still